CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+### Adds
+
+- Adds additional color options to `Icon` component.
```

Comment:
Add that the type/array structure was updated as well to make it easier to update.